### ISSUE TITLE
docker broke something

### ISSUE THREAD

**Issue by plusk01** [mit-acl/sfpro-dev#5]
```bash
$ ./build.sh --bash
docker: Error response from daemon: no status provided on response: unknown.
```

from @aleixpb2

**Comment by aleixpb2** [comment]
Solved when moving the project to a path with no spaces

**Comment by plusk01** [comment]
```bash
plusk01 ~/dev/snapdragon_flight/excelsior/sfpro (master) $ ./build.sh --bash
docker: Error response from daemon: no status provided on response: unknown.
plusk01 ~/dev/snapdragon_flight/excelsior/sfpro (master) $ pwd
/home/plusk01/dev/snapdragon_flight/excelsior/sfpro
```
I still get the error and I don't have spaces...

Does this mean that installing a specific version of `runc` was the fix after all?

**Comment by plusk01** [comment]
Latest updates fixes this on Ubuntu 16.04

```bash
$ uname -a
Linux lusk31 4.15.0-62-generic #69~16.04.1-Ubuntu SMP Fri Sep 6 02:43:35 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux
```

```bash
$ docker --version
Docker version 19.03.4, build 9013bf583a
```

```bash
$ containerd --version
containerd containerd.io 1.2.10 b34a5c8af56e510852c35414db4c1f4fa6172339
```

```bash
$ runc --version
runc version 1.0.0-rc8+dev
commit: 3e425f80a8c931f88e6d94a8c831b9d5aa481657
spec: 1.0.1-dev
```
